assets/js/core/WakaTimeCore.js

```diff
@@ -63,14 +60,11 @@ class WakaTimeCore {
             url: config.currentUserApiUrl,
             dataType: 'json',
             success: (data) => {
-
                 deferredObject.resolve(data.data);
-
+                this.recordHeartbeat();
```

Comment:
Let's move this to `assets/js/components/WakaTime.jsx` after line 69, to keep the init logic in one place.